### PR TITLE
setup.py

### PR DESCRIPTION
**Pull request by sfllaw**
I've fixed issue #2 by creating a setup.py

I've also renamed http_domain to sphinx_http_domain, to make it less likely to conflict with something else and to match the package name.

Finally, I've uploaded to PyPi. Please give me your PyPi username and I'll make you an owner of the package.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+setup(
+    name='sphinx-http-domain',
+    version='0.1',
+    description='Sphinx domain to mark up RESTful web services in ReST',
+    url='https://github.com/deceze/Sphinx-HTTP-domain/',
+    author='David Zentgraf',
+    author_email='deceze@gmail.com',
+    py_modules=['sphinx_http_domain'],
+    requires=['Spinx'],
```
Comment (deceze) [inline setup.py:11]: Ain't there a typo here?

**Comment by deceze** [comment]
My PyPi username is deceze.

**Comment by deceze** [comment]
Thank you for setting this up properly! :)

File: setup.py (hunk 1 above)
Comment (sfllaw) [inline setup.py:11]: Yes, you're right! Thanks for fixing it.

**Comment by sfllaw** [comment]
No problem. It's been pulled.

**Comment by sfllaw** [comment]
I mean, you've been added as an admin.

**Pull request merged**